src/common.cpp

```diff
@@ -31,22 +31,44 @@
 
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
+
+
+
+bool is_broadcastable(std::vector<std::size_t> s0,
```

Comment:
This should be removed and its unnecessary for this issue.

Reply:
done.